lib/tasks/source_file.rake

```diff
@@ -0,0 +1,17 @@
+namespace :source_file do
+  task create_imports: :environment do
+    count = 0
+    SourceFile.missing_import.find_each do |source_file|
+      import = source_file.create_import!
+
+      if import
+        count += 1
+
+        if count % 200 == 0
+          puts "Record count: #{index + 1}"
+          sleep 10
```

Comment:
@littleforest same here. Not sure if this was only for debugging

Reply:
Just wanted to add a little sleep since we have a fair number of source files.